target/linux/ipq40xx/patches-5.4/0017-v5.8-phy-add-driver-for-Qualcomm-IPQ40xx-USB-PHY.patch

```diff
@@ -137,15 +124,6 @@ Signed-off-by: John Crispin <john@phrozen.org>
 +	reset_control_deassert(phy->srif_rst);
 +	msleep(10);
 +
-+	writel(PHY_CTRL0_VAL, phy->base + PHY_CTRL0_ADDR);
-+	writel(PHY_CTRL1_VAL, phy->base + PHY_CTRL1_ADDR);
-+	writel(PHY_CTRL2_VAL, phy->base + PHY_CTRL2_ADDR);
-+	writel(PHY_CTRL3_VAL, phy->base + PHY_CTRL3_ADDR);
-+	writel(PHY_CTRL4_VAL, phy->base + PHY_CTRL4_ADDR);
-+	writel(PHY_MISC_VAL, phy->base + PHY_MISC_ADDR);
-+	writel(PHY_IPG_VAL, phy->base + PHY_IPG_ADDR);
```

Comment:
What happened to the magic values?

Reply:
They are not needed at all, they were only for preproduction HW.

Sartura has been upstreaming all of this throughout the last and this year as we use mainline kernels.
So, everything is being used actively.